stock_picking_send_by_mail/data/stock_picking_mail.xml

```diff
@@ -24,7 +26,6 @@
     <p style="border-left: 1px solid #8e0000; margin-left: 30px;">
        &nbsp;&nbsp;<strong>REFERENCES</strong><br />
        &nbsp;&nbsp;Picking number: <strong>${object.name}</strong><br />
-       &nbsp;&nbsp;Picking date: ${format_tz(object.date, tz=user.tz, context={'lang':object.partner_id.lang})} <br />
```

Comment:
If this date is still needed, it could be used `format_datetime`, since `format_tz` doesn't exist now in v13.0. 
https://github.com/odoo/odoo/blob/40c80961a87ea3c3c1a6f5ec798b842bd6256375/addons/mail/models/mail_template.py#L278

Reply:
Yes...
Here an example...
https://github.com/odoo/odoo/blob/13.0/addons/event/data/email_template_data.xml#L38